share/ruby-install/truffleruby-graalvm/functions.sh

```diff
@@ -37,37 +44,46 @@ function install_ruby()
 		return 1
 	fi
 
-	log "Installing GraalVM $ruby_version ..."
-	copy_into "$ruby_build_dir" "$install_dir/graalvm" || return $?
+	log "Installing TruffleRuby GraalVM $ruby_version ..."
+	if (( truffleruby_major >= 23 )); then # 23.1+
```

Comment:
Actually this `if` will be taken for version `23.0.0` which is not wanted.
Any idea how I could get the minor component of the version in Bash?

Reply:
I found a way